cpp/src/arrow/compute/kernel_test.cc

```diff
@@ -476,26 +459,27 @@ TEST(KernelSignature, MatchesInputsWithConstraint) {
   auto small_scale_decimal = decimal128(precision, small_scale);
   auto big_scale_decimal = decimal128(precision, big_scale);
```

Comment:
Perhaps the test would be more interesting if those types had different precisions?

Reply:
This is not directly related to this PR though, updated by more interesting combinations of (p, s).

Reply:
Thank you.